Schema_refinery/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Top-level package for Schema Refinery."""
+
+__author__ = """UMMI"""
+__email__ = 'microbiologia@fm.ul.pt'
```

Comment:
Not sure if this is the email. @ramirma, should we switch to [imm-bioinfo@medicina.ulisboa.pt](mailto:imm-bioinfo@medicina.ulisboa.pt)?